dashboard/test/models/script_test.rb

```diff
@@ -10,6 +10,13 @@ def setup
     Rails.application.config.stubs(:levelbuilder_mode).returns false
   end
 
+  def populate_cache_and_disconnect_db
+    Script.stubs(:should_cache?).returns true
+    Script.script_cache_to_cache
+    Script.script_cache_from_cache
+    ActiveRecord::Base.connection.disconnect!
```

Comment:
Is there any reason we shouldn't be disconnecting the DB rather than just the Script model?